.travis/scripts.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
```

Comment:
Well since this only runs on Travis it's no biggie. The moment somebody wants to run this script locally it is because the location of `bash` is dependant on the OS. Here's a good [stackoverflow answer](http://stackoverflow.com/a/10383546).